src/math/max.js

```diff
@@ -0,0 +1,18 @@
+const max = (numArray) => {
+let currentHighNumber = 0;
```

Comment:
To include negative numbers I would initially set `currentHighNumber = -Infinity`. 
What would your function return if you passed in this array:
```
let allNegativeArray = [ -1, -4, -90, -4 ]
```